packages/opentelemetry-core/package.json

```diff
@@ -78,7 +78,7 @@
     "karma-mocha": "2.0.1",
     "karma-spec-reporter": "0.0.36",
     "karma-webpack": "4.0.2",
-    "lerna": "6.0.3",
+    "lerna": "6.6.2",
```

Comment:
Why is this here? I would expect lerna only in root.

Reply:
It looks like this was added in #2484, back then there was a discussion about why this was added, and the reason for it was that the package could not be built in isolation without it. I guess the same would be true for all of our packages (`lerna` is used in the precompile script of every package). 

I checked some of the history, and I wondered why we don't run the version script with node directly (as we do with `update-ts-configs.js`). It seems to me that we would be able to simplify the whole thing. It looks like this was introduced in #2431. I am aware this was a very long time ago: @dyladan do you remember why we don't just run the `version` script directly in packages? :thinking:

Reply:
Invoking directly seems to work fine. Opened a PR to change this #3887

Reply:
Looks like this is here as `lerna` is used by the `precompile` script. 
https://github.com/open-telemetry/opentelemetry-js/pull/3887#issuecomment-1589303564

It was initially intended to have it in `devDependencies` in every package, as originally mentioned in https://github.com/open-telemetry/opentelemetry-js/pull/2484#discussion_r716038687. Looks like there was never a follow-up to that action item. I opened a PR add it everywhere (which is the opposite of what I wanted but oh well :sweat_smile:) #3909